lib/requests/diaryFoods.js

```diff
@@ -31,12 +31,13 @@ const populateMealTable = (meal) => {
 }
 
 const renderFoodToMealTable = (meal, food) => {
-  $(`#${meal.name.toLowerCase()}-table-info`).append(
+  var mealName = meal.name ? meal.name.toLowerCase() : meal
```

Comment:
I like the use of this ternary operator to make this method more flexible and able to handle different types of data being passed in

Reply:
🎉 🎉 🎉